src/callbacks/live_mode.py

```diff
@@ -96,18 +343,122 @@ def toggle_controls(n_clicks):
         allow_duplicate=True,
     ),
     Input("go-live", "n_clicks"),
+    State("selected-live-models", "data"),
     prevent_initial_call=True,
 )
-def update_data_project_dict(n_clicks):
+def update_data_project_dict(n_clicks, selected_models):
```

Comment:
You had commented before that this change may conflict with [PR#41](https://github.com/mlexchange/mlex_latent_explorer/pull/41). I think we should merge your PR first, and adapt PR 41 to accomodate the State of `selected-live-models` in the client-side callback